S7.Net/Types/DWord.cs

```diff
@@ -12,36 +12,39 @@ public static class DWord
         /// </summary>
         public static UInt32 FromByteArray(byte[] bytes)
         {
-            return FromBytes(bytes[3], bytes[2], bytes[1], bytes[0]);
+            return (UInt32)(bytes[0] << 24 | bytes[1] << 16 | bytes[2] << 8 | bytes[3]);
         }
 
+
         /// <summary>
-        /// Converts a S7 DWord (4 bytes) to uint (UInt32)
+        /// Converts 4 bytes to DWord (UInt32)
         /// </summary>
-        public static UInt32 FromBytes(byte v1, byte v2, byte v3, byte v4)
+        public static UInt32 FromBytes(byte b1, byte b2, byte b3, byte b4)
```

Comment:
Maybe more obvious naming is better when talking about bytes in a DWord/Word, starting at byte 0.
`public static UInt32 FromBytes(byte b0, byte b1, byte b2, byte b3)`